test/tools/ossfuzz/abiV2ProtoFuzzer.cpp

```diff
@@ -50,11 +74,37 @@ DEFINE_PROTO_FUZZER(Contract const& _input)
 		// We always call the function test() that is defined in proto converter template
 		hexEncodedInput = methodIdentifiers["test()"].asString();
 	}
-	catch (...)
+	catch (Exception const&)
 	{
-		cout << contract_source << endl;
-		throw;
+		return;
 	}
-	// TODO: Call evmone wrapper here
-	return;
+
+	// We target the default EVM which is the latest
+	langutil::EVMVersion version = {};
+	auto hostContext = EVMHost(version, evmone);
+	dev::bytes input = fromHex(hexEncodedInput);
+	evmc_message message;
+	message.kind = EVMC_CALL;
```

Comment:
Shouldn't it be create?

Reply:
I am executing the runtime bytecode of the factory contract. iiuc, create is used to first deploy the "deploy bytecode" of the contract, that essentially deploys the runtime bytecode at an address. Subsequently, we do issue a "call" on a function of the deployed contract at the deployed address.

I bypass this somewhat elaborate setup, by simply executing the runtime bytecode of the factory contract and calling the test function. The EVMHost implementation is used to handle the state required by the runtime bytecode.

Reply:
Since we have this interface now, maybe it would be better to make it proper. So do not use the factory contract but instead just create `C` and call its `test` function in two steps.

Reply:
Sure, I'll change the proto converter template like wise.